dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -2228,20 +2218,12 @@ private void populateDatasetUpdateFailureMessage() {
             // that must have been a bulk file update or delete:
             if (bulkFileDeleteInProgress) {
                 JsfHelper.addFlashErrorMessage(BundleUtil.getStringFromBundle("dataset.message.bulkFileDeleteFailure"));
-
             } else {
                 JsfHelper.addFlashErrorMessage(BundleUtil.getStringFromBundle("dataset.message.filesFailure"));
             }
         } else {
-
-            if (editMode.equals(EditMode.CREATE)) {
-                JsfHelper.addFlashErrorMessage(BundleUtil.getStringFromBundle("dataset.message.createFailure"));
-            }
-            if (editMode.equals(EditMode.FILE)) {
-                JsfHelper.addFlashErrorMessage(BundleUtil.getStringFromBundle("dataset.message.filesFailure"));
-            }
+            JsfHelper.addFlashErrorMessage(BundleUtil.getStringFromBundle("dataset.message.createFailure"));
```

Comment:
I understand that edit mode in this place is always in create mode?

Reply:
Yes, what's left is editMode == null and editMode == 'CREATE'/